.abs/main.yaml

```diff
@@ -0,0 +1,4 @@
+replace-chart-version-with-git: true
+generate-metadata: true
+chart-dir: ./helm/descheduler
+destination: ./build
```

Comment:
Please, add a new line at the end of your files. It's happening on several of them.